runtime/src/tests/functionalTests/page_objects/planner/planner.page.ts

```diff
@@ -9,7 +9,8 @@ export class PlannerPage extends AppPage {
   quickAdd =  new planner.WorkItemQuickAdd($('alm-work-item-quick-add'));
   sidePanel = new planner.SidePanel($('aside.f8-sidepanel'));
   quickPreview = new planner.WorkItemQuickPreview($('work-item-preview'));
-
+  header = new planner.ToolbarHeader($('pfng-toolbar'));
+  settings = new planner.Settings($('div.f8-wi-list__settings'));
   constructor(url: string){
     super();
     this.url = url;
```

Comment:
We should change the constructor. Can you please change 
```
 constructor(url: string){
     super();
     this.url = url;
```
to
```
 constructor(url: string){
     super(url);
```

Reply:
done

Reply:
protractor config updated please check